srcpkgs/libshumate/template

```diff
@@ -1,21 +1,22 @@
 # Template file for 'libshumate'
 pkgname=libshumate
-version=1.1.3
+version=1.2.0
 revision=1
 build_style=meson
 build_helper="gir"
 configure_args="$(vopt_bool gir vapi) $(vopt_bool gir gir) $(vopt_bool gir gtk_doc)"
```

Comment:
add `--buildtype debugoptimized` to `configure_args` to fix tests.

Reply:
ty great call